index.html

```diff
@@ -2,15 +2,46 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8" />
+    
+    <link rel="stylesheet" href="styles/style.css">
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Instagram Clone</title>
     <!-- don't forget to link your styles -->
+
+  
   </head>
-  <body>
-    Instagram Capture and Share the World's Moments Instagram is a fast, beautiful and fun way to share your life with
-    friends and family. Take a picture or video, choose a filter to transform its look and feel, then post to Instagram
-    &mdash; it's that easy. You can even share to Facebook, Twitter, Tumblr and more. It's a new way to see the world.
-    Oh yeah, did we mention it's free? Download on the App Store Get it on Google Play About Us Support Blog Press API
-    Jobs Privacy Terms &copy; 2014 Instagram
+  <body >
+    <div class="flex-container">
+      
+      <div>
+        <img src="../lab-css-instagram-clone/images/phones.png" alt="phones">
+      </div>
+
+      <div >
+        <div class="flex-container3">
+        <div class="logo"> <img src="../lab-css-instagram-clone/images/brand.png" alt="Instagram "></div>
+        <div class="login"> <img src="../lab-css-instagram-clone/images/home.png" alt="Instagram "> <a href="#" class= "login2"> Log in</a></div></div>
```

Comment:
Aunque las imagenes se verían en tu ordenador lo que estás haciendo es salir de la carpeta para volver a entrar en ella, esto aparte de ineficiente no funcionaría en un servidor real o en otro ordenador.
```suggestion
        <div class="logo"> <img src="./images/brand.png" alt="Instagram "></div>
        <div class="login"> <img src="./images/home.png" alt="Instagram "> <a href="#" class= "login2"> Log in</a></div></div>
```
Con el `./` indicamos que parte del directorio actual.